### ISSUE TITLE
Tree Crashes when there are too many objects

### ISSUE THREAD

**Issue by denizul** [ditrit/OGrEE-CLI#55]
Tree crashes due to 'too many open socket files'

**Comment by hchybz** [comment]
- works on WSL
- works on WSL

-

 Works oin WSL 
- Crash with even no <depth> on windows 

```
C:\Users\ziad\Desktop\OGrEE>main-compiled-wsl-2022-10-06.exe -e env.txt
Unity Client is Reachable!
Received from Unity : Connected to API
ziad@OGrEE3D:/Physical$> ls
Stray
DEMO
Billy
ziad@OGrEE3D:/Physical$> tree DEMO
/Physical/DEMO
panic: runtime error: index out of range [1] with length 1

goroutine 1 [running]:
cli/controllers.tree({0xc0001746c0, 0xe}, 0x0)
        /OGrEE/OGrEE-CLI/controllers/commandController.go:1194 +0x226
cli/controllers.Tree({0xc0001746a0, 0xe}, 0x0)
        /OGrEE/OGrEE-CLI/controllers/commandController.go:1180 +0x1b2
main.(*treeNode).execute(0xc000004210)
        /OGrEE/OGrEE-CLI/ast.go:577 +0x53
main.(*ast).execute(0xc000004228)
        /OGrEE/OGrEE-CLI/ast.go:38 +0x69
main.InterpretLine(0xc000110370)
        /OGrEE/OGrEE-CLI/repl.go:31 +0xc2
main.Repl(0xc00018e1f8, {0xc000010a45, 0x4})
        /OGrEE/OGrEE-CLI/repl.go:212 +0x105
main.Start(0x465660)
        /OGrEE/OGrEE-CLI/repl.go:196 +0x5aa
main.main()
        /OGrEE/OGrEE-CLI/main.go:109 +0x187e

C:\Users\ziad\Desktop\OGrEE>main-compiled-wsl-2022-10-06.exe -e env.txt
Unity Client is Reachable!
Received from Unity : Connected to API
```

**Comment by hchybz** [comment]
`git  checkout master / git pull / make win `

essai sur c.api. tree EDF 15 
ok pour moi
- CLI/Win + API/c.api : OK
- CLI/Win + API WSL : OK
- CLI/WSL + API c.api : OK
- CLI/WSL + API WSL : OK

**Comment by denizul** [comment]
Issue is resolved as of commit on master branch: f693467
